src/utils/sellerStats.js

```diff
@@ -0,0 +1,33 @@
+const stats = (arr, numSales) => {
+  const sum = arr
+    .map((product) => {
+      return Number(product.price);
+    })
+    .reduce((accumulator, currentValue) => accumulator + currentValue, 0);
+
+  const expiredProducts = arr
+    .map((p) => {
+      const now = Date.now();
+      const d = new Date(p.expiryDate);
```

Comment:
use clear variable declarations.

Eg: instead of `...map((p)=>.... `
Use `...map((product)=>....`

`const d = new Date(...)   -------> const expiryDate = new Date(...)`